### PR TITLE
fix simple typos

### PR DESCRIPTION
**Pull request by lpmi-13**

### CODE REVIEW

**Comment by lpmi-13** [comment]
since this is just the README, I thought it would be easier to target the `master` branch, but if you would prefer this PR to target `dev` branch, just let me know.

File: README.md [hunk 1]
```diff
@@ -15,21 +15,21 @@
 
 [用 Rails 开发 Go 应用：中文文档](./README_zh.md)
 
-`go-on-rails` is a Rails generator aims to:
+`go-on-rails` is a Rails generator which aims to:
 
-1. Help to develop and integrate some APIs written in Golang to existed Rails app for high performance
-2. Use your farmiliar Rails tools to develop and manage a Golang app project
+1. Help to develop and integrate some APIs written in Golang to existing Rails app for high performance
+2. Use your familiar Rails tools to develop and manage a Golang app project
```
Comment (gingerhot) [inline README.md:21]: yes, `farmiliar` is a typo, and `existed` should be `existing`.

File: README.md [hunk 2]
```diff
@@ -93,9 +93,9 @@ rails g gor --help
 
 * Go project directory layout (all under the `go_app` directory, like `views`, `controllers`, `public`)
 * A Go data struct corresponding to each activerecord model
-* And each struct related CRUD functions/methods like FindModel, UpdateModel, DestroyModel etc. All these models related program files under the `go_app/models` directory
+* And each struct related CRUD functions/methods like FindModel, UpdateModel, DestroyModel etc. All the related program files for these models under the `go_app/models` directory
```
Comment (gingerhot) [inline README.md:96]: don't change.

File: README.md [hunk 3]
```diff
@@ -93,9 +93,9 @@ rails g gor --help
...
 * And godoc files for all the functions under `go_app/models/doc`
-* We use [Gin](https://github.com/gin-gonic/gin) as the default web framework, but you can change it to anyone that you favor in `main.go` and `controllers` programs
+* We use [Gin](https://github.com/gin-gonic/gin) as the default web framework, but you can change it to any other one that you favor in `main.go` and `controllers` programs
```
Comment (gingerhot) [inline README.md:98]: don't change.

File: README.md [hunk 4]
```diff
@@ -50,7 +50,7 @@ $ gem install go-on-rails
 ```
 ## Usage
 
-You must have an existed Rails app or to create a new one before you try go-on-rails to generate Golang codes.
+You must have an existing Rails app or create a new one before you try go-on-rails to generate Golang codes.
```
Comment (gingerhot) [inline README.md:53]: don't need change.

File: README.md [hunk 5]
```diff
@@ -15,21 +15,21 @@
...
 * A simple [example(tutorial)](https://github.com/railstack/example_simple) on the basic usage of go-on-rails generator
 * [An advanced example](https://github.com/railstack/example_with_admin) shows how to integrate Go APIs in a Rails project
-* [Another example](https://github.com/railstack/example_read_rails_session) shows how to handle a Rails session to get an user's info in a go-on-rails generated Go API
+* [Another example](https://github.com/railstack/example_read_rails_session) showing how to handle a Rails session to get a user's info in a go-on-rails generated Go API
 
 ## Prerequisites
 
 * Rails 4.2+ (Rails 6 not supported and need your help)
-* Golang 1.10.x(mainly be compatible with github.com/go-sql-driver/mysql)
+* Golang 1.10.x(mainly compatible with github.com/go-sql-driver/mysql)
```
Comment (gingerhot) [inline README.md:32]: don't change.

File: README.md [hunk 6]
```diff
@@ -15,21 +15,21 @@
...
-2. Use your farmiliar Rails tools to develop and manage a Golang app project
+1. Help to develop and integrate some APIs written in Golang to existing Rails app for high performance
+2. Use your familiar Rails tools to develop and manage a Golang app project
 3. Convert a *not very complicated* Rails app to Golang equivalent
 
 Here's some examples:
 * A simple [example(tutorial)](https://github.com/railstack/example_simple) on the basic usage of go-on-rails generator
 * [An advanced example](https://github.com/railstack/example_with_admin) shows how to integrate Go APIs in a Rails project
-* [Another example](https://github.com/railstack/example_read_rails_session) shows how to handle a Rails session to get an user's info in a go-on-rails generated Go API
+* [Another example](https://github.com/railstack/example_read_rails_session) showing how to handle a Rails session to get a user's info in a go-on-rails generated Go API
```
Comment (gingerhot) [inline README.md:27]: keep the `shows`.

File: README.md [hunk 7]
```diff
@@ -15,21 +15,21 @@
 
 [用 Rails 开发 Go 应用：中文文档](./README_zh.md)
 
-`go-on-rails` is a Rails generator aims to:
+`go-on-rails` is a Rails generator which aims to:
```
Comment (gingerhot) [inline README.md:18]: don't change. just keep it short.

**Comment by gingerhot** [comment]
hi, I really appreciate the great job you did, and I give some comments, I would merge the PR after you revise it. Thanks again!

**Comment by lpmi-13** [comment]
yep, I'm happy to revise as you requested.

**Comment by lpmi-13** [comment]
alright, should be updated to reflect the requested changes. Please let me know if anything else should be updated. Awesome project btw, looking forward to playing with it!

**Pull request merged**